src/run.rs

```diff
@@ -12,7 +12,7 @@ use nu_protocol::{engine::StateWorkingSet, Module, PipelineData, ShellError, Spa
 use nu_utils::utils::perf;
 
 fn get_standard_library() -> &'static str {
-    include_str!("../crates/nu-utils/standard_library/std.nu")
+    include_str!("../crates/nu-std/std.nu")
 }
```

Comment:
this would need to live in the `nu-std` crate and imported here via `nu_std::get_standard_library()`

Reply:
yup that's exactly what i have in mind :yum: